app/services/utils.py

```diff
@@ -0,0 +1,164 @@
+        pass
+    else:
+        # `script` is a byte string, decode it using UTF-8
+        script = script.decode('utf-8')
+
+    newScript = script.replace("-"," ")
+    for charToRemove in TO_REMOVE:
+        newScript = newScript.replace(charToRemove,"")
+
+    while "<" in newScript:
+        start = newScript.index("<")
+        end = newScript.index(">")+1
+        newScript = newScript[:start]+newScript[end:]
+    while "  " in newScript:
+        newScript = newScript.replace("  "," ")
+    while "\n " in newScript:
+        newScript = newScript.replace("\n ","\n")
+    while " \n" in newScript:
+        newScript = newScript.replace(" \n","\n")
+    while newScript[0] == " ":
+        newScript = newScript[1:]
+
+    return newScript
+
+
+def creat_randome_name():
+    # Define the length of the random string
+    length = 12
+    while True:
+        # Generate a random string of length 8
+        random_string = ''.join(random.choices(string.ascii_letters + string.digits, k=length))
+        # Check if a file with that name already exists in the directory
+        if not os.path.exists(random_string):
+            break
+    # Use the unique random string for the file name
+    filename = random_string 
+
+    return filename
+
+
+
+def save_audio(audio_data, file_path, file_name):
+    """
+    Save an audio file to a specified path with a given filename.
+
+    Args:
+    - audio_data: bytes, the binary data of the audio file to be saved
+    - file_path: str, the path to the directory where the audio file should be saved
+    - file_name: str, the name of the audio file
+
+    Returns:
+    - None
+    """
+
+    # Concatenate the file path and file name to create the full file path
+    full_path = file_path + file_name+'.wav'
+
+    # Load the audio data into an AudioFileClip object
+    # Write the audio data to a file
+    with open(full_path, 'wb') as f:
+        f.write(audio_data)
+
+
+
+def delete_cache():
+    for filename in os.listdir():
+        # Check if the file ends with '_final.mp4'
+        if filename.endswith('_final.mp4'):
+            file_path = os.path.join(os.getcwd(), filename)
+            try:
+                # Check if the file is in use
+                with open(file_path, 'r'):
+                    pass
+                # If not in use, delete the file
+                os.remove(file_path)
+                print(f"{filename} deleted successfully.")
+            except PermissionError:
+                print(f"{filename} is in use. Skipping deletion.")
+            except Exception as e:
+                print(f"Error deleting {filename}: {e}")
+
+
+
+
+def get_video_from_file(path):
+    # Read the video file as binary data
+        def iterfile():  # 
+             with open(path+'_final.mp4', mode="rb") as file_like:  # 
+                yield from file_like  # 
+
+        return StreamingResponse(iterfile(), media_type="video/mp4")
+
+
+
+def delete_temprory_files(folder_path, prefix):
```

Comment:
Fix typo

Reply:
For lazyKh orginal code will be done after having a second working version